### DISCUSSION TITLE
Accessibility 101

### DISCUSSION THREAD

**Discussion by anthonyking2112** [community/community#31733] [Accessibility]
What is the shortcut too access menu

**Comment by benjiallen** [comment]
@anthonyking2112 what menu are you trying to access? If you could let me know the URL and describe the feature you're trying to access then I should be able to help.

**Comment by cehfisher** [comment]
Hi @anthonyking2112 - was your issue resolved? It looks like Ben did not have enough information to follow-up on your request. This may be a stale issue we need to close @queenofcorgis
